eq-author/src/App/page/Design/answers/BasicAnswer/index.js

```diff
@@ -150,9 +153,14 @@ export const StatelessBasicAnswer = ({
           if (error.errorCode === "PIPING_TITLE_DELETED") {
             message = PIPING_TITLE_DELETED.message;
           }
+          if (error.errorCode === "PIPING_TITLE_MOVED") {
+            message = PIPING_TITLE_MOVED.message;
+          }
           return (
             error.field === "label" && (
-              <ValidationError key={error.id}>{message}</ValidationError>
+              <ValidationError key={error.id === "PIPING_TITLE_MOVED"}>
```

Comment:
Not sure why this has been added. This is either not required, or it needs to be set to the correct value at all times. It can't be hardcoded to PIPE_TITLE_MOVED when the message can be from a different error entirely.